src/ansys/motorcad/core/rpc_client_core.py

```diff
@@ -413,7 +414,8 @@ def _open_motor_cad_local(self):
             )
 
         motor_process = subprocess.Popen(
-            [self.__MotorExe, "/PORT=" + str(self._port), "/SCRIPTING"]
+            [self.__MotorExe, "/PORT=" + str(self._port), "/SCRIPTING"],
+            cwd=Path(self.__MotorExe).parent.absolute(),
```

Comment:
This sets current working directory to the parent of the Motor-CAD exe. Same as if you launched it by clicking the icon